libs/csrf/csrfprotector.php

```diff
@@ -104,11 +104,22 @@ public static function init($length = null, $action = null)
 			if (session_id() == '')
 			    session_start();
 
-			if (!file_exists(__DIR__ ."/../config.php"))
-				throw new configFileNotFoundException("OWASP CSRFProtector: configuration file not found for CSRFProtector!");	
+			/*
+			 * load configuration file and properties
+			 * Check locally for a config.php then check for 
+			 * a config/csrf_config.php file in the root folder
+			 * for composer installations
+			 */
+			$standard_config_location = __DIR__ ."/../config.php";
+			$composer_config_location = __DIR__ ."/../../../../../config/csrf_config.php";
```

Comment:
That is a relative path to a config/ folder in a local project folder after installing via the library via composer.